src/init.cpp

```diff
@@ -453,6 +459,7 @@ std::string HelpMessage(HelpMessageMode mode)
     if (showDebug)
     {
         strUsage += HelpMessageOpt("-logtimemicros", strprintf("Add microsecond precision to debug timestamps (default: %u)", DEFAULT_LOGTIMEMICROS));
+        strUsage += HelpMessageOpt("-mocktime=<n>", "Replace actual time with <n> (default: 0)");
```

Comment:
It's a developer documentation and I think it's useful to know that 0 means noop.

Though, I am using your extended description.